docs/source/contributing/accessibility-guidelines.md

```diff
@@ -9,16 +9,30 @@ myst:
 
 # Accessibility guidelines
 
-Attention should be paid to improve and maintain accessibility. In documentation, code and filenames you will often see this abbreviated as "a11y".
+Attention should be paid to improve and maintain accessibility, abbreviated as "a11y".
 
-## Clickable elements that have a symbol or icon and no visible text
 
-If the symbol or icon is clear enough for sighted users, for instance a big "X" to close a dialog, it will still need a text for screen reader users.
+## Tests
 
-You can achieve a 'hidden' label by setting the property "aria-label" on the element.
-If available, use the translation machinery available to make the label appear in the correct language.
+We use [`cypress-axe`](https://github.com/component-driven/cypress-axe) for automated testing of accessibility in Volto via Cypress.
 
-Example:
+Place accessibility test files in the subfolder in the Volto package.
+For example, `packages/volto/cypress/tests/core/blocks/blocks-a11y.js`.
```

Comment:
@stevepiercy I think I did a suggestion in a PR yesterday. We could create a folder `a11y` on its own in `packages/volto/cypress/tests/core/a11y` or even as a first level test type, and create a separated Cypress process in GHA in order to fail quickly on these.

Also, some a11y needs setup, so it makes sense that eventually, it uses a different fixture for the test at some point.

We normally do this in our projects and works well.

Reply:
This PR is the outcome of a discussion amongst the a11y team. They thought that perhaps developers would remember to work on a11y tests if they were in the same location as other tests for the object being tested. I think both ideas have advantages and disadvantages. I have no opinion, other than it gets documented. Let's see what other members of the @plone/volto-accessibility team think.

Reply:
> @stevepiercy I think I did a suggestion in a PR yesterday. We could create a folder `a11y` on its own in `packages/volto/cypress/tests/core/a11y` or even as a first level test type, and create a separated Cypress process in GHA in order to fail quickly on these.
> 
> Also, some a11y needs setup, so it makes sense that eventually, it uses a different fixture for the test at some point.
> 
> We normally do this in our projects and works well.

@stevepiercy @sneridagh I'm fine also to create a new folder in core named a11y and there we have:
- blocks.js
-  content.js
-  views.js

I agree also on the quick failure possibly running before the regular cypress tests.
As long as we have some separation I am flexible on how we achieve this and will go with the
decision of the majority :)

Reply:
Given that the a11y Team was not certain about the location, and we just kinda picked one to move ahead, and now we have a motion and second for a different location, I'll make the change to the docs.

When the CI items are put into practice, we can document them.